### DISCUSSION TITLE
Mobile engineering performance, productivity, and maturity benchmark survey

### DISCUSSION THREAD

**Discussion by RikH81** [MobileNativeFoundation/discussions#161] [General]
Hi folks 👋,

We (at Bitrise) see tens of thousands of mobile teams on our platform, and looking at all of those teams, the feeling has always been that there's very little process standardisation or shared understanding of what is "good". To help go from feeling to fact, we put together a survey: MODAS, short for Mobile DevOps Performance, Productivity, and Maturity Assessment (excuse the clumsy abbreviation - it's the best thing we could come up with after we figured that MDOPPMA wouldn't work)

We now have 400+ teams that have submitted performance benchmarks, listed practices, and gave some more subjective judgments on their maturity in the areas of:

- Creation
- Testing
- Deploying
- Monitoring
- Collaborating

When we hit a more statistically relevant number (2.5k+), we'll turn this into a live assessment tool and benchmark report that will show how industries, technologies, and team sizes compare.

If you have 5-10 minutes to spare, it'd be a massive help if you could fill out that survey here > https://survey.alchemer.eu/s3/90442780/Mobile-DevOps-Performance-Productivity-and-Maturity-Assessment

Thanks, and happy building!

**Comment by shahzadmajeed** [comment]
Could you make this assessment public?

**Reply by mkattima1** [reply]
Hi @RikH81 , do we have this assessment available now?

**Comment by RikH81** [comment]
@shahzadmajeed @mkattima1 
Sorry for the amount of time this is taking - We just published the first part of the data, specifically for apps in the finance and banking industry [here](https://yo.bitrise.io/rs/700-EMV-600/images/MODAS-report.pdf) (it's a gated download, but that link will take you directly to the PDF).

We'll speak more about the data at the [Mobile DevOps Summit](https://www.mobiledevops.io/summit/home) in November, plus - by then - hope to have published additional insights

**Reply by mkattima1** [reply]
Thanks @RikH81 , this is very helpful

**Reply by shahzadmajeed** [reply]
@RikH81 thank you for the document. Looking forward to the summit as well :)

**Reply by mikktammeglia** [reply]
Hello,

Interesting report. But how do you look at the feature delivery speed. Features can be very different size. Does this not influence the answer very much?
